src/installer/tests/HostActivation.Tests/StartupHooks.cs

```diff
@@ -147,16 +147,20 @@ public void Muxer_activation_of_RuntimeConfig_And_Environment_StartupHooks_Succe
             var startupHook2Fixture = sharedTestState.StartupHookWithDependencyFixture.Copy();
             var startupHook2Dll = startupHook2Fixture.TestProject.AppDll;
 
-            // RuntimeConfig and Environment startup hook
+            const string whitespacePattern = @"[\r\n\s]*";
+
+            // RuntimeConfig and Environment startup hooks in expected order
             dotnet.Exec(appDll)
                 .EnvironmentVariable(startupHookVarName, startupHook2Dll)
                 .CaptureStdOut()
                 .CaptureStdErr()
                 .Execute()
                 .Should().Pass()
-                .And.HaveStdOutContaining("Hello from startup hook!")
-                .And.HaveStdOutContaining("Hello from startup hook with dependency!")
-                .And.HaveStdOutContaining("Hello World");
+                .And.HaveStdOutMatching("Hello from startup hook with dependency!" +
+                                        whitespacePattern +
+                                        "Hello from startup hook!" +
+                                        whitespacePattern +
+                                        "Hello World");
```

Comment:
This is unfortunately not 100% reliable.
We've seen tests fail because there are multiple threads running at the same time (bread-crumb writer) which can also print to output.
See https://github.com/dotnet/runtime/issues/57243. I don't know 100% if that can happen here as well... but I'd rather make the test resilient to that.

We could probably change the regex to accept any text in between to "fix" this.

Reply:
I have to apologize - I should have mentioned this problem beforehand... sorry.

Reply:
ah np I understand. Thanks for letting me know.

Reply:
I have slightly changed the pattern and added a comment.